config/locales/en.yml

```diff
@@ -3,7 +3,7 @@ en:
     statuses:
       draft: Draft
       submitted_for_2i: Submitted for 2i
-      in_review: In review
+      in_review: In 2i review
```

Comment:
I'd tweak the commit message (`shouldn't be really clear on the value of that before proceeding` should be _should_, I think?)

Reply:
Thanks 👍 good catch.